### PR TITLE
Fix ogcproxy_enable settigs and update the documentation

### LINKED ISSUE #1282: disable OGC proxy gives an error
related to the commit https://github.com/camptocamp/c2cgeoportal/commit/a6fe1285daa5fa911f772f127f66ce8c70c59866

following the doc in
http://docs.camptocamp.net/c2cgeoportal/1.5/integrator/security.html?highlight=ogc#enable-disable-the-ogc-proxy

```ogcproxy_enable: false``` gives the following error:

```
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]     return fix_call(context.object, context.global_conf, **context.local_conf)
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]   File "/home/vvmalbre/geoview/buildout/eggs/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]     val = callable(*args, **kw)
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]   File "/home/vvmalbre/geoview/bl_gis2012/__init__.py", line 113, in main
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]     config.include('c2cgeoportal')
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]   File "/home/vvmalbre/geoview/buildout/eggs/pyramid-1.5.1-py2.7.egg/pyramid/config/__init__.py", line 755, in include
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]     c(configurator)
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]   File "/home/vvmalbre/geoview/buildout/eggs/c2cgeoportal-1.5.3-py2.7.egg/c2cgeoportal/__init__.py", line 218, in includeme
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199]     if config.get_settings().get("ogcproxy_enable", "true").lower() in ["true", "yes", "1"]:
[Tue Mar 03 16:25:18 2015] [error] [client 192.168.25.199] AttributeError: 'bool' object has no attribute 'lower'
```

also, these options dont work:

- ```ogcproxy_enable: False``` -> same error as above
- ```ogcproxy_enable: 'false'``` -> viewer.js does not work

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1282

### CODE REVIEW

File: doc/integrator/security.rst [hunk 1]
```diff
@@ -35,3 +35,8 @@ Enable / Disable the OGC proxy
 To disable the OGC proxy, set ``ogcproxy_enable`` to ``false`` in your ``config.yaml.in``.
 
 Default: ``true``
+
+In the ``viewer.js`` files you also should remove the ``OpenLayers.ProxyHost`` configuration.
```
Comment (asaunier) [inline doc/integrator/security.rst]: you also should => you should also

File: doc/integrator/security.rst [hunk 2]
```diff
@@ -35,3 +35,8 @@ Enable / Disable the OGC proxy
...
+
+This implies the all the external WMS (from the database and from the WMS browser) should
```
Comment (asaunier) [inline doc/integrator/security.rst]: This implies that all external WMS services

File: doc/integrator/security.rst (hunk 1 above)
Comment (sbrunner) [inline doc/integrator/security.rst]: Fixed :-)

File: doc/integrator/security.rst (hunk 2 above)
Comment (sbrunner) [inline doc/integrator/security.rst]: fixed :-)

**Comment by asaunier** [comment]
Looks good.

**Pull request merged**